### PR TITLE
Readme guide

### PR DESCRIPTION
**Pull request by jennhurn**

### CODE REVIEW

File: ottos_expeditions/README.md [hunk 1]
```diff
@@ -0,0 +1,539 @@
...
+  The query aggregates expedition data to determine the success rate of expeditions based on the route and its difficulty level. This analysis helps us understand which routes are more challenging and how frequently expeditions successfully reach the summit.
+
+### 3. Modify the Materialization Type
+
+Let's say we want to modify the materialization type of the `route_difficulty_success_prediction_transform.sql.jinja` component to a view. We can do this by changing the materialization type to `view` in the `route_difficulty_success_prediction_transform.sql.jinja` file.
+
+Here's how to do it:
+
+- Click on the `route_difficulty_success_prediction_transform.sql.jinja` file to open it.
+- Change the materialization type to `view` by adding CODE to the code #hi Tessa :)
```
Comment (jennhurn) [inline ottos_expeditions/README.md:333]: @tessa0413  -- Not 100% sure how to do this

File: ottos_expeditions/README.md [hunk 2]
```diff
@@ -0,0 +1,539 @@
...
+
+  This query joins data from the three read components to analyze the impact of weather conditions on expedition outcomes. By calculating the `weather_impact_rate`, it provides insights into the percentage of expeditions that were affected by weather conditions.
+
+  One important difference between this flow and the previous ones is that we're using parameters to pass the `success_rate_threshold` to the flow at runtime (See line 21). Parameters allow us to pass variables to the flow, making it more flexible and reusable.
+
+### 3. Adjust Pipeline Parameters
+
+Let's say we want to adjust the pipeline parameters to enhance analysis precision. We can do this by changing the runtime parameters for the the `weather_impact_analysis_transform.sql.jinja` file.
+
+To do this, tessa is gunna explain how to do it. :) YAY TESSA
```
Comment (jennhurn) [inline ottos_expeditions/README.md:382]: @tessa0413 -- Also not sure how to do this :D

File: ottos_expeditions/README.md [hunk 3]
```diff
@@ -0,0 +1,539 @@
...
+### 3. Add Data Quality Tests
+
+Let's add a data quality tests to our `gear_rentals` component.
+
+  - To add a data quality tests, add the following code to the `gear_rentals.yaml` file:
+
+      ```yaml
+      tests:
+        columns:
+          Hi tessa :)
```
Comment (jennhurn) [inline ottos_expeditions/README.md:213]: hiiii @tessa0413 can you add the DQ checks here?

File: ottos_expeditions/README.md [hunk 4]
```diff
@@ -0,0 +1,539 @@
...
+### 3. Implement Data Partitioning on the Transformation Component
+
+Let's say we want to implement data partitioning to optimize the flow. The files we are ingesting in this flow are already partitioned by date, so we can use the partitions from upstream to partition data in the transformation component. 
+
+Here's how to do it:
+
+- Click on the `customer_journey_conversion_analysis_transform.sql.jinja` file to open it.
+- Add the following code to the file:
+
+    ```sql
```
Comment (jennhurn) [inline ottos_expeditions/README.md:429]: @tessa0413 - could you add the map partitions stuff here?

File: ottos_expeditions/README.md (hunk 1 above)
Comment (tessa0413) [reply ottos_expeditions/README.md:333]: done

File: ottos_expeditions/README.md (hunk 3 above)
Comment (tessa0413) [reply ottos_expeditions/README.md:213]: done

File: ottos_expeditions/README.md (hunk 4 above)
Comment (tessa0413) [reply ottos_expeditions/README.md:429]: done

File: ottos_expeditions/README.md [hunk 5]
```diff
@@ -0,0 +1,554 @@
...
+  - To add a data quality tests, add the following code to the `gear_rentals.yaml` file:
+
+      ```yaml
+      tests:
+        columns:
+          rental_date:
+            - not_null
+      ```
+
+  #Tessa - Also please add a quick description of the test.
```
Comment (jennhurn) [inline ottos_expeditions/README.md:217]:
```suggestion
 With this test, we're ensuring the `rental_date` column is not null.  
```

**Review by ayushsm — COMMENTED** [review]
Just wanted to check on the Python file, everything else seems close to fine. Will hold off on merge until Shayaan pushes smaller CSV files and Jon fixes commit history.

**Comment by sixdimensionalarray** [comment]
Merged manually to `hack-marquee-project`, closing PR

**Pull request closed without merging**